google/cloud/automl/v1beta1/automl_gapic.yaml

```diff
@@ -347,6 +402,8 @@ interfaces:
       name: dataset
     resource_name_treatment: STATIC_TYPES
     timeout_millis: 5000
+   # This is also broken
```

Comment:
What's broken? :)

Reply:
They are all fixed , ignore all these broken comments.


Han helped me to fix them, just forgot to delete them afterwards.

Reply:
Remove the comments? :P